src/quad/CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ foreach(SIMD ${SLEEFQUAD_SUPPORTED_EXT})
   endif()
 endforeach()
 
-if(MSVC OR MINGW AND WIN32)
+if(MSVC OR SLEEF_CLANG_ON_WINDOWS OR MINGW AND WIN32)
```

Comment:
Why is this condition different from the one `if((MSVC OR MINGW AND WIN32) OR SLEEF_CLANG_ON_WINDOWS)` 25 of `src/libm/CMakeLists.txt`?

Reply:
Those two are now same.